src/codemagic/models/certificate.py

```diff
@@ -52,6 +52,13 @@ def from_ans1(cls, asn1: AnyStr) -> Certificate:
         x509_certificate = cls._get_x509_certificate(asn1, crypto.FILETYPE_ASN1)
         return Certificate(x509_certificate)
 
+    @classmethod
+    def from_p12(cls, p12: bytes, password: Optional[AnyStr] = None) -> Certificate:
+        encoded_password = password and password.encode(encoding='utf-8') or None
```

Comment:
It does not work with `bytes` input right now:
```python
password = b'a'
password and password.encode(encoding='utf-8') or None
Traceback (most recent call last):
  File "/Users/priit/.virtualenvs/builder/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 3417, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-31-28414d572b07>", line 1, in <module>
    password and password.encode(encoding='utf-8') or None
AttributeError: 'bytes' object has no attribute 'encode'
```
Why don't use want to use the properly tested convenience method from `StringConverterMixin`?

Reply:
It is also detected by type checks: https://github.com/codemagic-ci-cd/cli-tools/pull/74/checks?check_run_id=1986298776#step:9:7